### ISSUE TITLE
Enhancement: show username and hostname

### ISSUE THREAD

**Issue by wsh3776** [dracula/tmux#82]
### Describe the feature you'd like
Is it possible to add a setting that can show the current `username` and `hostname`

### Describe risks you've considered
I think it wouldn't have much risk. 

### Describe alternatives you've considered
I can use `echo user` and `hostname` to show this, but can I show it in the bottom status bar. I often login the remote machine. So if I can see which host I am current in, it would be nice!

![image](https://user-images.githubusercontent.com/46078304/97103215-f9406180-16e5-11eb-9cdf-ec83069f3b7b.png)

I think it can be achieved by edit the `script.sh`, but I am not sure how to do it.

**Comment by ethancedwards8** [comment]
Yeah we could do this, but I don't really see how useful it is. You could literally just type `whoami` and `hostname` and get those. We need to reduce feature bloat because now this project is becoming quite large and we should focus on the things that are actually needed. Like working with operating systems on than Linux and Mac, better compatibility with other plugins, color theming, etc.

Closing unless I can get a really good reason on why we need these. :)

**Comment by qifei9** [comment]
I work on my laptop and a remote machine connect through ssh from my laptop at the same time. Thus I often have multiple tmux session running, some locally and some remotely. Therefore, I hope the status bar could tell me where the session is running, locally or remotely. Although I could have these info in the zsh prompt, the prompt is not always visible, for example in vim.

**Comment by osbm** [comment]
What @qifei9 describes is exactly why i need this feature. Is there a way around to do this?

**Comment by osbm** [comment]
Can we reopen this issue? @ethancedwards8

**Comment by Theoreticallyhugo** [comment]
using `set -g @dracula-show-left-icon "#H"` one can get the host name of the machine where the session runs in the top left. what i find even more useful is `set -g @dracula-show-left-icon "#H | #S"` which tells me on which machine i am and what the name of the session is, since some of my servers have multiple sessions for different long run projects.

i think that with this in mind, we should be able to close this issue again. mby i should write about this in the docs #278

**Comment by osbm** [comment]
@Theoreticallyhugo your answer is very interesting i will try that out.

I should mention, i was able to meet my need with just by doing this:
```
set -g @dracula-show-left-icon hostname
```
